crates/red_knot_python_semantic/resources/mdtest/attributes.md

```diff
@@ -1209,6 +1209,17 @@ class C:
 reveal_type(C().x)  # revealed: Unknown
 ```
 
+### Accessing attributes on `Never`
```

Comment:
Is it worth also explicitly testing that arbitrary attributes can be set without any errors?

Reply:
> that arbitrary attributes can be set without any errors

Is that the case? Never can be assigned to anything, but not the other way around. So I would expect every attribute assignment to be an error, unless the rhs-type is `Never`.

Reply:
> unless the rhs-type is `Never`

Yes, it was this that I was thinking of

Reply:
(It might be worth adding a test for it either way, if we don't currently have any)